### PR TITLE
DOCSP-22867: describe ChangeStreamDocument

### PR DESCRIPTION
**Pull request by rustagir**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-java/blob/master/REVIEWING.md)

JIRA - [DOCSP-22867](https://jira.mongodb.org/browse/DOCSP-22867)
[Staging](https://docs-mongodbcom-staging.corp.mongodb.com/java/docsworker-xlarge/DOCSP-22867-describe-changestreamdocument/fundamentals/crud/read-operations/change-streams/#example)

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [ ] Did you run a grammar-check?
- [ ] Are all the links working? NO LINKS

### CODE REVIEW

File: source/fundamentals/crud/read-operations/change-streams.txt [hunk 1]
```diff
@@ -85,6 +86,14 @@ following text:
       ...
    }
 
+.. note:: ChangeStreamIterable and ChangeStreamDocument Types
+   
+   The driver stores change stream events in a variable of type
+   ``ChangeStreamIterable``. In the preceding example, the driver stored
+   individual change stream events as ``ChangeStreamDocument`` objects,
+   since we specified that the driver should populate the ``ChangeStreamIterable``
+   with ``Document`` types.
```
Comment (DBirtolo-mdb) [inline source/fundamentals/crud/read-operations/change-streams.txt:95]: [Suggestion]

A couple of things here. First, the sentence is a little long, so I would recommend breaking it up. Also, I would put the action that we did first, to put the focus on that, and then have the effect later. Something like the following:

```suggestion
   ``ChangeStreamIterable``. In the preceding example, we specified the driver should
   populate the ``ChangeStreamIterable`` with ``Document`` types. As a result, the driver
   stores the individual change stream events as ``ChangeStreamDocument`` objects.
```

**Review by DBirtolo-mdb — CHANGES_REQUESTED** [review]
Nice work. A couple of small things that occurred to me but only one approval-blocking suggestion.

File: source/fundamentals/crud/read-operations/change-streams.txt [hunk 2]
```diff
@@ -63,8 +63,9 @@ changes in the connected MongoDB deployment.
...
 ~~~~~~~
 
-The following code example shows how to open a change stream and print
-change stream events whenever the data in the collection changes:
+The following code example shows how to open a change stream on a
+collection and print change stream events whenever the data in the
+collection changes:
 
 .. literalinclude:: /includes/fundamentals/code-snippets/change-streams/ChangeStreams.java
    :language: java
```
[the remark this answers is not in the document]
Comment (rustagir) [reply source/fundamentals/crud/read-operations/change-streams.txt:71]: The way the code is set up right now, a collection is passed as a parameter. I'm going to edit this to instantiate the collection inside the function though.

File: source/fundamentals/crud/read-operations/change-streams.txt [hunk 3]
```diff
@@ -85,6 +86,14 @@ following text:
       ...
    }
 
+.. note:: ChangeStreamIterable and ChangeStreamDocument Types
```
[the remark this answers is not in the document]
Comment (rustagir) [reply source/fundamentals/crud/read-operations/change-streams.txt:89]: Agreed - but what do you mean about changing the introduction?

**Review by DBirtolo-mdb — APPROVED** [review]
LGTM!

File: source/fundamentals/crud/read-operations/change-streams.txt (hunk 3 above)
[the remark this answers is not in the document]
Comment (DBirtolo-mdb) [reply source/fundamentals/crud/read-operations/change-streams.txt]: Oh - I just meant the wording would need to be reworked. It's all good!

**Pull request merged**